content/javascript/concepts/objects/terms/create/create.md

```diff
@@ -0,0 +1,47 @@
+Title: 'Object.create() in JavaScript'
+Description: 'static method creates a new object, using an existing object as the prototype of the newly created object.It basically allows you to create a new object that inherits properties and methods from a specified prototype object. It's not about making a copy of an existing object, but rather establishing a prototype chain.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+  - 'javascript'
+  - 'objects'
+  - 'prototype'
+   Tags:
+  - 'javascript'
+  -  'objects'
+  - 'inheritance'
+  - 'prototype'
+  - 'Documentation'
+  CatalogContent:
+  - 'javascript-objects'
+  - 'paths/computer-science'
+
+  ## Syntax
+ ```js
+Object.create(proto)
+Object.create(proto, propertiesObject)
+ ```
+  The **Object.create()** method creates a new object with the specified prototype object and properties. If no prototype is provided, the newly created object will have null as its prototype. This means that the newly created object will not inherit any properties or methods from any other object.
```

Comment:
This is the description of the Object.create() method, this should go at the start, before the ## Syntax. 
Please modify the line: " The **Object.create()** method creates a new object with the specified prototype object and properties", since it is plagiarised.